dags/bqetl_artifact_deployment.py

```diff
@@ -76,7 +76,7 @@
             "script/bqetl query schema update '*' --use-cloud-function=false --ignore-dryrun-skip --project-id=moz-fx-data-marketing-prod && "
             "script/bqetl query schema deploy '*' --use-cloud-function=false --force --ignore-dryrun-skip --project-id=moz-fx-data-marketing-prod"
```

Comment:
Is `marketing-prod` missing an `&&` at the end?

Reply:
ah, good catch